### ISSUE TITLE
GitHub/Git panel re-opens when Atom application window is moved

### ISSUE THREAD

**Issue by danchristian** [atom/github#2282]
### Prerequisites

* [ X ] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Steps to Reproduce

1. Open Git or GitHub panel via footer buttons
2. Close panel via footer button
2. Move Atom application window

**Expected behavior:**

The panel doesn't re-open

**Actual behavior:**

The panel re-opens

**Reproduces how often:**

100%

### Versions

```
MacOS Mojave 10.14.6

Atom    : 1.40.1
Electron: 3.1.10
Chrome  : 66.0.3359.181
Node    : 10.2.0
```

### Additional Information

https://www.dropbox.com/s/53f4o6pv9ud9061/atom.mov?dl=0

**Comment by rsese** [comment]
Thanks for the report!  I was unable to reproduce on macOS 10.14.6 with 1.40.1.  Did you see the same behavior when you tried in safe mode?

**Comment by danchristian** [comment]
Yes. You can view the initial history of me reporting the bug and trying safe mode etc [here](https://atomio.slack.com/archives/C044E54H0/p1569593092070600).

**Comment by rsese** [comment]
> Yes. You can view the initial history of me reporting the bug and trying safe mode etc [here](https://atomio.slack.com/archives/C044E54H0/p1569593092070600).

ahh great, thanks for that context :+1:  Not sure why I'm not seeing it but it looks like Ash can reproduce.

**Comment by thirstyice** [comment]
Could be related to #1650
